client/src/components/CardFilter.tsx

```diff
@@ -444,6 +488,36 @@ export function CardFilter(props: {
     { id: 'only', name: 'Only' },
   ]
 
+  const actionOptions: { id: string; name: string; searchText: string; icon?: string }[] = [
+    { id: 'action:', name: 'Action', searchText: 'action:' },
```

Comment:
The searchText "action:" will also find "Reaction:". 

searchTexts should be used case sensitive

Reply:
So we should explicitly search for "Action: (or \<b>Action:\</b>, even)

Reply:
Fixed.